src/__tests__/helpers/auth-test-helpers.ts

```
@@ -0,0 +1,223 @@
+    updateProfile: vi.fn(),
+    refreshSession: vi.fn(),
+    ...overrides
+  }
+}
+
+export async function waitForAuthStateChange() {
+  // Wait for auth state changes to propagate
+  await new Promise(resolve => setTimeout(resolve, 100))
+}
+
+export function expectToastMessage(toastMock: any, type: 'success' | 'error', message?: string) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)